ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,29 @@
+ages = [29, 19, 17, 26, 32, 16, 21]
+print(ages)
+print(ages[4:5])
+print(ages[-2:-1])
+print(ages[3:6])
```

Comment:
This range starts at the fourth element, not the third:

---

![image](https://user-images.githubusercontent.com/16143771/149258129-4929c44c-a707-463b-86d6-6b1f51bd4d73.png)